bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/handler/SomfyTahomaBridgeHandler.java

```diff
@@ -396,6 +408,34 @@ private void stopPolling() {
         return null;
     }
 
+    public @Nullable List<SomfyTahomaDevice> getDevices() {
```

Comment:
Would it be possible to return `Collections.emptyList()`? Then you don't need to do a null-check on the result

Reply:
ok